src/components/QuickLinksDialog/index.tsx

```diff
@@ -0,0 +1,98 @@
+import GordonDialogBox from 'components/GordonDialogBox';
+import LinkIcon from '@mui/icons-material/InsertLink';
+import { ListItemIcon, ListItemText, ListSubheader, List, ListItem } from '@mui/material';
+import CanvasIcon from './images/Canvas.ico';
+import GordonIcon from './images/favicon.ico';
+import GOIcon from './images/GoGordonFavicon.ico';
+import MyGordonIcon from './images/MyGordonFavicon.ico';
+import { PropsWithChildren } from 'react';
+
+type Props = {
+  handleLinkClose: () => void;
+  linkopen: boolean;
+};
+
+const customIconLinks: ListItemProps[] = [
+  {
+    href: 'https://www.gordon.edu',
+    name: 'Gordon College',
+    icon: {
+      src: GordonIcon,
+      alt: 'gordon',
+    },
+  },
+  {
+    href: 'https://my.gordon.edu',
+    name: 'My Gordon',
+    icon: { src: MyGordonIcon, alt: 'my gordon' },
+  },
+  { href: 'https://go.gordon.edu', name: 'Go Gordon', icon: { src: GOIcon, alt: 'go gordon' } },
+  { href: 'https://canvas.gordon.edu', name: 'Canvas', icon: { src: CanvasIcon, alt: 'canvas' } },
+];
+
+const otherLinks = [
+  {
+    href: 'http://athletics.gordon.edu/',
+    name: 'Fighting Scots',
+  },
+  { href: 'http://stories.gordon.edu/', name: 'The Bell' },
+  { href: 'https://www.gordon.edu/titleix', name: 'Sexual Discrimination and Harassment' },
+  { href: 'https://www.gordon.edu/map', name: 'Gordon College Maps' },
+];
+
+const GordonQuickLinksDialog = ({ linkopen, handleLinkClose }: Props) => {
+  return (
+    <GordonDialogBox
+      aria-labelledby="useful-links"
+      open={linkopen}
+      title="Useful Links"
+      buttonClicked={handleLinkClose}
+      buttonName="Close"
+    >
+      <List
+        component="nav"
+        subheader={
+          <ListSubheader component="div" disableSticky>
+            Academics
+          </ListSubheader>
+        }
+      >
+        {customIconLinks.map((link) => (
+          <ListItemLink {...link} />
+        ))}
+      </List>
+      <List component="nav" subheader={<ListSubheader component="div">Information</ListSubheader>}>
```

Comment:
`Information` isn't technically true as far as how we describe the links in the code above. Perhaps we should do away with both headers (`Academics` and `Information`) and just have a list of links?

Or it should be `Academics` and `Other` and we should update the code to follow that structure.

It doesn't really make sense in my mind to have two different arrays of links dependent on whether they have an icon or not. That can just be an optional property with a default value. It **would** make sense to have two separate arrays if we split it by academic and other.